applications/service_brokers/installing-service-catalog.adoc

```diff
@@ -0,0 +1,12 @@
+[id='sb-installing-service-catalog']
+= Installing the service catalog
+include::modules/common-attributes.adoc[]
+:context: sb-installing-service-catalog
+
+toc::[]
+
+IMPORTANT: The service catalog is deprecated in this release. Equivalent and better functionality is present in the Operator Framework and
```

Comment:
Probably check with @mpbarrett or @robszumski on the exact wording here (ie Equivalent and better...)  I just want to be certain we are putting out the right message.  We use this term in the brokers and catalog section, so once you have the right text we'll want to update all occurrences.

Reply:
@jboyd01, @chengzhang1016:

@mpbarrett actually provided the wording to me for the deprecation notice for the release notes, see here: http://file.rdu.redhat.com/~ahardin/02252019/4-0-release-notes/release_notes/ocp-4-0-release-notes.html#ocp-4-0-notable-technical-changes

I pulled just part of this in to the deprecation notice in the actual content here. Let me know if this wording needs to be updated at all here or in the release notes. Thanks!